pkg/common/utils/utils.go

```diff
@@ -104,12 +104,21 @@ func IsAssignedPod(pod *v1.Pod) bool {
 }
 
 func GetQueueNameFromPod(pod *v1.Pod) string {
```

Comment:
The order to get 'queue' from pod:

Before this PR:
1. Label: constants.LabelQueueName
2. Annotation: constants.AnnotationQueueName
3. Default: constants.ApplicationDefaultQueue

After this PR
**1. Label: constants.CanonicalLabelQueueName** (New)
2. Label: constants.LabelQueueName
3. Annotation: constants.AnnotationQueueName
4. Default: constants.ApplicationDefaultQueue